### PR TITLE
#659  Implement min price for Category, Series and OptionsQS

### LINKED ISSUE #659: Implement min price for category, series, product pages
Then use it at the page templates like this: `Цены от: [минимальная цена] руб`.
See the details at https://github.com/fidals/stroyprombeton/issues/653

### PR DESCRIPTION
**Pull request by duker33**
Closes #659

### CODE REVIEW

File: stroyprombeton/models.py [hunk 1]
```diff
@@ -192,6 +206,10 @@ def tagged_or_all(self, tags: 'TagQuerySet'):
             else self
         )
 
+    def min_price(self) -> float:
+        min_price_qs = self.annotate(min_price=models.Min('price'))
```
Comment (artemiy312) [inline stroyprombeton/models.py]: It seems it won't find min price
```
In [19]: P.objects.aggregate(min_price=Min('price'))
Out[19]: {'min_price': 11.0}

In [20]: P.objects.annotate(min_price=Min('price')).first().price
Out[20]: 1300.0

In [21]: P.objects.order_by('price').first().price
Out[21]: 11.0
```

File: stroyprombeton/tests/tests_models.py [hunk 2]
```diff
@@ -1,16 +1,37 @@
...
 
+from pages import models as pages_models
 from stroyprombeton import models as stb_models
 
 
 @tag('fast')
-class Category_(TestCase):
+class Option_(TestCase):
 
     fixtures = ['dump.json']
```
Comment (artemiy312) [inline stroyprombeton/tests/tests_models.py]: It seems you should remove this, because you create custom data for this test case

**Comment by duker33** [comment]
seems i've lost `Category_.test_series` test. I'll return it back

**Comment by duker33** [comment]
[ci answered "ok"](https://ci.fidals.com/fidals/stroyprombeton/1062), but Github don't see the result.
[Webhook work good](https://ci.fidals.com/fidals/stroyprombeton/1062) too, so it's Github problems, i suppose.

Merge it

**Pull request merged**